install_files/securedrop-config/DEBIAN/control

```diff
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: SecureDrop Team <securedrop@freedom.press>
 Homepage: https://securedrop.org
 Package: securedrop-config
-Version: 0.1.2+0.13.0~rc1
+Version: 0.1.3+0.13.0~rc1
```

Comment:
The version bump here is absolutely correct. It requires matching changes in other places, though:

* group_vars for staging (already handled)
* config tests vars (specifically, see `molecule/builder-trusty/tests/vars.yml`)

The rationale here is that we want to ensure we're installing _exactly_ a certain version, and also running the package checks on _exactly_ that same version. By default, multiple package versions can pile up in `build/`. We could automatically run `rm -rf build/*` to avoid the pile-up, but that seems a bit heavy-handed for the dev env.

Reply:
Fixed `config_version` in vars.yml.